interface-impl-amf/pom.xml

```diff
@@ -18,8 +18,8 @@
         <formatterConfigPath>../formatter.xml</formatterConfigPath>
         <sonar.jacoco.reportPaths>../target/jacoco.exec</sonar.jacoco.reportPaths>
 
-        <amf.version>4.0.6</amf.version>
-        <amf.xml.extension.version>1.4.1</amf.xml.extension.version>
+        <amf.version>4.1.0-2-INTERNAL-SNAPSHOT</amf.version>
```

Comment:
Can we use syncValidate in XML payloads ? using this AMF version

Reply:
extension should be 1.6.0

Reply:
sure `<amf.xml.extension.version>1.6.0</amf.xml.extension.version>`

Reply:
my comment was in the previous line , sorry 😆

Reply:
will do !